dcc-portal-ui/app/scripts/app/js/app.js

```diff
@@ -469,24 +470,40 @@
       }]);
 
     })
-    .run(function($state, $location, $window, $timeout, $rootScope, cfpLoadingBar, HistoryManager, gettextCatalog, Settings) {
+    .run(function($state, $location, $window, $timeout, $rootScope, cfpLoadingBar, HistoryManager, gettextCatalog, Settings, REDIRECTS) {
       
       // Setting the initial language to English CA.
       gettextCatalog.setCurrentLanguage('en_CA');
 
+      const redirect = _.find(REDIRECTS, (redirect) => redirect.from === $location.url());
```

Comment:
This looks to be in the wrong scope. Won't this capture the url at loading time?